HIPs/incubation/identus.md

```diff
@@ -0,0 +1,172 @@
+
+---
+layout: default
+title: Identus
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+Hyperledger Identus v1.0 (was Open Enterprise Agent lab)
+
+# Sponsor(s)
+- Jon Bauer - jon@goodfuture.io
+- Lance Byrd - lance.byrd@rootsid.com
+- Roberto Carvajal - roberto@goodfuture.io  
+- Stephen Curran - swcurran@cloudcompass.ca
+- Esteban Garcia - esteban.garcia@iohk.io
+- Mix Irving - mix@protozoa.nz  
+- Rodolfo Miranda - rodolfo.miranda@rootsid.com
+- Darrell O’Donnell - darrell.odonnell@continuumloop.com
+- Javier Ribo - javier.ribo@iohk.io
+- Björn Sandmann - sandmann@codedata.solutions
+- Yurii Shynbuiev - yurii.shynbuiev@iohk.io
+- Lohan Spies - lohan.spies@iohk.io
+- Ben Tairea - ben@ahau.io
+- Benjamin Voiturier - benjamin.voiturier@iohk.io
+- Marcus Ubani - marcus@larissa.health  
+
+# Abstract
+Identus provides components to develop decentralized identity solutions that adhere to widely recognized self-sovereign identity (SSI) standards. 
```

Comment:
As a comparison
> Hyperledger Aries is your complete toolkit for decentralized identity solutions and digital trust. Issue, store and present verifiable credentials with maximum privacy preservation, and establish confidential, ongoing communication channels for rich interactions. It supports multiple protocols, credential types, ledgers and registries. It has frameworks in multiple development languages, and interoperability tools and profiles to help everything work together seamlessly.

Distinguishing between Aries and Identus could be a challenge. What are the reasons for not bringing the code base into Aries? This is something that we will need answered in this proposal (not in the abstract, but this is something that I will be looking for as I review this proposal).

Reply:
Identus was conceived as a standalone set of tools. It effectively takes from Aries specs, but it wasn't designed as an extension to Aries functionalities but rather an implementation of the specs using different technologies. Additionally, Aries and Identus are not interoperable, mainly due to a difference in the DIDComm version. Aries uses DIDComm v1, while Identus only supports DIDComm V2. I must highlight that there may be additional complexities in achieving compatibility, but the DIDComm versioning is the first hurdle. 

So, our viewpoint is that it will be confusing to have components under the Aries umbrella that are not compatible with the rest of the Aries code base, while the Identus set of components works in conjunction as it is; therefore, It would be more logical to consider Identus as a standalone project.